### PR TITLE
CONTACT-form pull

### PR DESCRIPTION
**Pull request by Ramaunspoken**
Created the CONTACT-form and run the checks and then fixed the errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -73,6 +76,237 @@ <h3 class="social_title blue">LET’S CONNECT</h3>
...
+            />
+            <textarea
+              class="form_text_area"
+              name="message"
+              placeholder="Write your message here"
+              maxlength="500"
+              required
+            ></textarea>
+            <button type="submit" class="btn_submit">Get It Touch</button>
+          </fieldset>
```
Comment (omarsalem7) [inline index.html]: - [ ] Kindly make sure to match your design with [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) and fix the alignment of text and fix also the background to match the one in Figma.

| Your design  | Figma design|
| ------------- | ------------- |
| image | image |.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi team 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by LethaboMot** [comment]
I also took part in the changes of the portfolio

File: index.html [hunk 2]
```diff
@@ -73,6 +76,237 @@ <h3 class="social_title blue">LET’S CONNECT</h3>
...
+          </h3>
+        </ul>
+      </section>
+      <section class="contact_form">
+        <h2 class="title">Contact me</h2>
+        <p class="contact_form_des">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
```
Comment (leonardodiasb) [inline index.html]: - [ ] As the previous code reviewer mentioned, please work on the style of your form heading and paragraph in order to match with the Figma template design. Kindly center the text, add some margin to the top and on the sides, and add the font family used in the whole template.
 -Your design:
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Ramaunspoken, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - There is no linter error present ✔️
 - The Pull Request has a proper title and description. ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sajid-munawar — APPROVED** [review]
### Hi @Ramaunspoken , 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
 ## [Optional] suggestions:
 * [ ]  Please check my inline comments.

> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — COMMENTED** [review]
Hi @Ramaunspoken , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle: :red_circle: 

You have submitted a project, that is already merged and approved.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**